CMakeLists.txt

```
@@ -1,11 +1,6 @@
-cmake_minimum_required(VERSION 3.11.0)
+cmake_minimum_required(VERSION 3.15.0)
 project(python-blosc2)
-find_package(PythonExtensions REQUIRED)
-find_package(Cython REQUIRED)
-find_package(NumPy REQUIRED)
+find_package(Python COMPONENTS Interpreter NumPy Development.Module REQUIRED)
```

```suggestion
find_package(Python 3.10 COMPONENTS Interpreter NumPy Development.Module REQUIRED)
```
This is a tricky part, but if you don't specify the minimum version here, it would not consider `python3` when looking for the executable. This is problematic since Fedora does not include a `python` symbolic link to `python3`.